### ISSUE TITLE
How to disable the restore workspace on quit?

### ISSUE THREAD

**Issue by ruudk** [TablePlus/TablePlus#1288]
With the latest update, TablePlus restores previously opened connections on launch. The first time it said how to disable this but I didn't read it. But now I look for that option in the preferences but cannot find an option to turn this off.

I don't want to restore my connections when I restart TablePlus.

**Comment by huyphams** [comment]
It's here @ruudk 
<img width="612" alt="aa" src="https://user-images.githubusercontent.com/3654622/60771133-9d4d6600-a0b9-11e9-8082-61a9160b0308.png">t

**Comment by ruudk** [comment]
@huyphams Thanks for your reply. 

There is a typo in the sentence: `exception WHEN you turn this option on`

Even when that typo is changed, it's still not clear to me.

I would rephrase it to this:

`[x] Restore previously opened workspaces on startup`
